lib/cc/analyzer/engine_output.rb

```diff
@@ -0,0 +1,33 @@
+module CC
+  module Analyzer
+    class EngineOutput
+      def initialize(raw_output)
+        @raw_output = raw_output
+      end
+
+      def blank?
+        raw_output.blank?
+      end
+
+      def issue?
+        parsed_output &&
+          parsed_output["type"].present? &&
+          parsed_output["type"].downcase == "issue"
+      end
+
+      def as_issue
+        Issue.new(raw_output)
```

Comment:
That starts to get into the validation territory IMO (if we're raising then we need to make sure it matches the errors builder expects, etc) - curious to see how this plays out as-is